routes/web/project/postgres.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class CloverWeb
+  hash_branch(:project_prefix, "postgres") do |r|
+    @serializer = Serializers::Web::Postgres
+
+    r.get true do
+      @postgres_databases = serialize(@project.postgres_databases_dataset.authorized(@current_user.id, "Postgres:view").all)
```

Comment:
It's beneficial to eagerly load `semaphores` and `strands`. Otherwise, it would result in 2N+1 requests.